src/styles/_base.scss

```diff
@@ -52,3 +52,11 @@ textarea:focus,
 button:focus {
   outline: none;
 }
+
+h3 {
```

Comment:
we have a texts.scss file for things like this as base should be to normalize the css.